modules/social_features/social_comment/social_comment.install

```diff
@@ -151,6 +151,11 @@ function social_comment_update_11201(): string {
 function social_comment_update_11202(): void {
   $comment_view = \Drupal::configFactory()->getEditable('views.view.comment');
 
+  // Do nothing if the comment view does not exist.
```

Comment:
@navneet0693  Could you explain your objection to `isNew()`, please? Since in that case we also will see either config already exist or not.

Reply:
@Ressinel Creating a configuration and then taking an action of `isNew()` seems redundant. Instead `\Drupal::configFactory()->get('views.view.comment');` this would not return anything if the configuration is not existing.

Reply:
@Ressinel We can improve the code if we think it adds value. Also, contributors will get credited once the PR is merged.

Reply:
Please also perform this step:

On a fresh Open Social or Drupal installation, delete this configuration. Now, add an update hook mimicking our view without the safety check and just call the `save` function. Go to the configuration table or use drush command to check if the configuration is there.

Reply:
@navneet0693 Isn't this what we are fixing? 😃

Reply:
@Ressinel I am unable to understand your comment. Please elaborate it.